navigator_engine/tests/integration_tests/test_end_to_end.py

```diff
@@ -7,7 +8,7 @@
 @pytest.mark.vcr()
 @pytest.mark.usefixtures('with_app_context')
 def test_end_to_end(client):
-    graph_loader('Estimates 22 BDG [Final].xlsx')
+    graph_loader(app.config.get('TEST_DATA_SPREADSHEET'))
```

Comment:
This test will only pass for the specific graph 'Estimates 22 BDG [Final].xlsx' so I would suggest explicitly mentioning the graph name here anyway, just for readability of the test.  i.e. suggestion is to leave this line as it was.